src/chrome/content/rules/ebayrtm.com.xml

```diff
@@ -0,0 +1,83 @@
+<!--
+
+	For other eBay coverage, see ebay.com.xml	
```

Comment:
Please also update the list in `ebay.com.xml`

Reply:
Added in https://github.com/EFForg/https-everywhere/pull/14626/commits/009a46792aa559d2146894613c875cff3bc78fd0